app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ReportingRepository.kt

```diff
@@ -320,13 +320,29 @@ class ReportingRepositoryImpl(
         }
     }
 
+    private suspend fun uploadRevocationTeks(
+        teks: List<TemporaryExposureKey>,
+        revoceWarningType: MessageType.InfectionLevel,
+        targetWarningType: MessageType
+    ) {
+        val validityIntervallsToRevoke = diagnosisKeysRepository
```

Comment:
We should put this work on a general purpose thread by wrapping it in `withContext(coroutineContext)`

Reply:
There is nothing blocking here. Looks to me like we can run on any thread and a context switch is more expensive then the work we are doing.